src/style.css

```
@@ -0,0 +1,100 @@
+/* Font Definitions */
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+/* Global Styles */
+:root {
+  --box-shadow-default: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  --box-shadow-hover: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  --box-shadow-focus: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  --font-color: #3d4e61;
+  --input-border: 1px solid #e1e7ed;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
```

```suggestion
* {
  box-sizing: border-box;
```
It is a good tone to change only box-sizing using universal selector